webAO/master.js

```diff
@@ -25,7 +25,7 @@ masterserver.onopen = (evt) => onOpen(evt);
 masterserver.onmessage = (evt) => onMessage(evt);
 
 const server_description = [];
-server_description[99] = "This is your computer on port 27016";
+server_description[-1] = "This is your computer on port 50001";
```

Comment:
so apparently how this works is that the negative indices get stored as keys, as if the array were an Object. it doesn't actually contribute to the array, since for all intents and purposes it's not part of the array...

i hate javascript lmao

Reply:
oof
do i change it or do we just live with it

Reply:
does it work right now?

Reply:
yes